providers/provider-pojo/src/main/java/org/apache/servicecomb/provider/pojo/Invoker.java

```diff
@@ -162,6 +162,8 @@ public Object invoke(Object proxy, Method method, Object[] args) {
 
     SwaggerConsumerOperation consumerOperation = pojoConsumerOperationMeta.getSwaggerConsumerOperation();
     OperationMeta operationMeta = pojoConsumerOperationMeta.getOperationMeta();
+    // operation meta is static data based on swagger, while consumer operation based on invoker, have interface info
+    operationMeta.setSwaggerConsumerOperation(consumerOperation);
```

Comment:
maybe there are different consumer interfaces.  
if set back to operationMeta, will cause problem.

Reply:
begin from scb-1706, that modify is not a good idea.

Reply:
From invocation, can only get OperationMeta. Current code put all invocation data(e.g. response type) to OperationMeta. This is not good, need refactor in future, but this PR not working on this refactoring, because it is related to some other parts besides this issue.  I create an [issue](https://issues.apache.org/jira/browse/SCB-1830) and refactoring the code later.

Reply:
as we disscussed, we create new issue to resolve this.